src/templates/base/2019/accessibility_statement.html

```diff
@@ -36,6 +36,11 @@
 {% block styles %}
 {{ super() }}
 <link rel="stylesheet" href="/static/css/page.css?v=20200603172138">
+<style>
+.year-switcher {
+  display: none;
```

Comment:
Similar to the 2020 styles, I think this is a job for the template.

Reply:
You've more of a case with this one alright! 😀

I did try to override the macro's (defined in `base.html`) in this `accessibility_statement.html` template but it wouldn't work. The accessibility statement is a special case - it's the only template which is language but not year specific. We've still decide how to handle this if 2019 and 2020 differ (ref: https://github.com/HTTPArchive/almanac.httparchive.org/pull/665#discussion_r379591735). So for now I'm happy with this but let's revisit once we have more decisions on 2019 and 2020.

And btw any thoughts on Methodology? Will that differ per year? As could be a lot of repetition on that too.

Reply:
Yes, Methodology pages will differ by year, although not by much. I'd like to see the methodologies as specific as possible for that year as a kind of "audit trail" in case anyone needed to see how we arrived at certain results. I fully expect there to be mostly minor differences year to year, but sometimes we may make more dramatic changes. In either case, we should make the doc as fully customized as possible. (I'll take ownership of that)

Reply:
OK. May need to be careful with Google seeing it as duplicate content.

Fixed the accessibility statement issue. Figured out [how to override macros in Jinja2](https://stackoverflow.com/questions/26582731/redefining-imported-jinja-macros).